src/detector/detector_app.py

```diff
@@ -33,18 +38,19 @@
 def create_app() -> FastAPI:
     detector_configuration_filepath: str = os.path.join(os.path.dirname(__file__), "..", "..", "data", "config.json")
     detector_configuration: DetectorConfiguration
-    #camera_interface: AbstractCameraInterface
+    camera_interface: AbstractCameraInterface
     marker_interface: AbstractMarkerInterface
 
     with open(detector_configuration_filepath, 'r') as infile:
         detector_configuration_file_contents: str = infile.read()
         detector_configuration_dict = hjson.loads(detector_configuration_file_contents)
         detector_configuration = DetectorConfiguration(**detector_configuration_dict)
 
-    #camera_interface = Detector.USBWebcamWithOpenCV()
+    camera_interface = PiCamera()
+    # camera_interface = USBWebcamWithOpenCV(detector_configuration.camera_connection.usb_id)
```

Comment:
I wonder if there's a cleaner way to pass an argument here to decide which camera to use